Doc/whatsnew/3.2.rst

```diff
@@ -1336,7 +1336,7 @@ Decimal('1.100000000000000088817841970012523233890533447265625')
 Fraction(2476979795053773, 2251799813685248)
 
 Another useful change for the :mod:`decimal` module is that the
-:attr:`Context.clamp` attribute is now public.  This is useful in creating
+:attr:`Context.clamp <decimal.Context>` attribute is now public.  This is useful in creating
```

Comment:
Public... but apparently still undocumented? Should this maybe be left as is, and added to this list for now?

https://github.com/python/cpython/blob/318f2190bc93796008b0a4241243b0851b418436/Doc/conf.py#L273-L279

Reply:
Sure, added!